axelrod/strategies/dbs.py

```diff
@@ -0,0 +1,354 @@
+from axelrod.actions import Actions, Action
+from axelrod.player import Player
+
+
+C, D = Actions.C, Actions.D
+
+def action_to_int(action):
+    return (1 if action==C else 0)
+
+
+class DBS(Player):
+    """
+    Desired Belief Strategy as described in:
+    Accident or Intention: That Is the Question (in the Noisy Iterated Prisoner's Dilemma) by Tsz-Chiu Au and Dana Nau from University of Maryland
```

Comment:
Sure, the full citation is:
T.-C. Au and D. S. Nau. Accident or intention: That is the question (in the iterated prisoner’s dilemma). In Proc. Int. Conf. Auton. Agents and Multiagent Syst. (AAMAS), pp. 561–568, 2006
I guess it becomes [Au2006] in the specific format ? I'll add it to the bibliography.rst file